packages/webdriverio/src/commands/element/addValue.js

```diff
@@ -20,22 +21,24 @@
  *
  * @alias element.addValue
  * @param {string | number | boolean | object | Array<any>}      value     value to be added
+ * @param {boolean} translateToUnicode enable translation string to unicode value automatically
```

Comment:
This needs to be formatted as follows so that type definitions are generated properly:

```suggestion
 * @param {AddValueOptions} options                    command options (optional)
 * @param {Boolean}         options.translateToUnicode enable translation string to unicode value automatically
```

Then please add the type to the [definition template](https://github.com/webdriverio/webdriverio/blob/master/scripts/templates/webdriverio.tpl.d.ts#L181):

```ts
interface AddValueOptions {
    translateToUnicode: boolean
}
```

Reply:
Fixed

Reply:
@christian-bromann , it seems that type should be added to some other files (at least `webdriverio-core-v5.d.ts`), isn't it?

Reply:
Something like this
```
addValue(
            value: string | number | boolean | object | any[],
            translateToUnicode?: AddValueOptions
        ): void;

```
?

Reply:
yeah